### PR TITLE
Correct GHA badge in README

### PR DESCRIPTION
**Pull request by gliptak**

### CODE REVIEW

**Comment by gliptak** [comment]
@glondu @rivy please review

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ Unison File Synchronizer
 ========================
 
 [![Build Status](https://travis-ci.org/bcpierce00/unison.svg?branch=master)](https://travis-ci.org/bcpierce00/unison)
-![Main workflow](https://github.com/bcpierce00/unison/workflows/Main%20workflow/badge.svg)
+[![Main workflow](https://github.com/bcpierce00/unison/workflows/Main%20workflow/badge.svg)](https://github.com/bcpierce00/unison/actions?query=workflow%3A+Main+Workflow")
```
Comment (gdt) [inline README.md:5]: Why is there a trailing "?  It seems spurious, even though I don't know the URL format, and if I remove it, the link still works.

Other than that looks good to me.

Comment (gliptak) [reply README.md:5]: I''m not sure which ? you refer to. The intended target is a filtered list.

Comment (gdt) [reply README.md:5]: I think we are having character set issues.    In the commit you want to merge, the URL for the workflow ends with the string Workflow followed by double-quote character.  There is no opening quote and I don't know why this one is there.

Comment (gliptak) [reply README.md]: updated. thx for correcting

**Pull request merged**